src/styles/blocks/catalog.scss

```
@@ -2,24 +2,32 @@
   display: flex;
   flex-wrap: wrap;
 
-  max-width: 944px;
+  max-width: $catalog-width;
   margin: 0 auto;
-  padding: 50px 0;
+  padding: $catalog-padding-y 0;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color $trans-duration $smooth,
+      transform $trans-duration $smooth;
```

my bad, I did not correct this name fully during the last review. When I speak of maximally clear variable names I mean that:
```suggestion
    transition: color $transition-duration $transition-timing-function,
      transform $transition-duration $transition-timing-function;
```

---

Ok, thanks, Ive got it.
This one is fixed.